lib/gems/pending/appliance_console/cli.rb

```diff
@@ -113,6 +113,7 @@ def parse(args)
         opt :username, "Database Username",  :type => :string,  :short => 'U', :default => "root"
         opt :password, "Database Password",  :type => :string,  :short => "p"
         opt :dbname,   "Database Name",      :type => :string,  :short => "d", :default => "vmdb_production"
+        opt :dedicated, "Configure this server as a dedicated database instance", :type => :bool, :short => 'D', :default => false
```

Comment:
Also, I don't think we should default this option. It will be true if it is specified otherwise `InternalDatabaseConfiguration` will deal with the default value. Similar to how we do `--key`